tests/integration/test_wazuh_db/test_wazuh_db.py

```diff
@@ -74,14 +126,18 @@ def clean_registered_agents():
 
 @pytest.fixture(scope='module')
 def wait_range_checksum_avoided(line):
-    """Callback function to wait until the manager avoided the checksum calculus by using the last saved one."""
+    '''
+    Callback function to wait until the manager avoided the checksum calculus by using the last saved one.
+    '''
     if 'range checksum avoided' in line:
         return line
     return None
 
 
 def wait_range_checksum_calculated(line):
-    """Callback function to wait until the manager calculates the new checksum."""
+    '''
+    Callback function to wait until the manager calculates the new checksum.
+    '''
```

Comment:
I would not change this line because it follows the Google style correctly, with a one-line summary. Also, try to use `"""` instead.
```suggestion
   """Callback function to wait until the manager calculates the new checksum."""
```

Reply:
Done in the following [commit](https://github.com/wazuh/wazuh-qa/commit/5ceb7bd05d809dc92ccde0df5d314a5914036df8)